content/en/learn.md

```diff
@@ -30,7 +30,7 @@ There is tons of information about NumPy out there. If you are new to NumPy, we'
 * [From Python to NumPy *by Nicolas P. Rougier*](https://www.labri.fr/perso/nrougier/from-python-to-numpy/)
 * [Elegant SciPy](https://www.amazon.com/Elegant-SciPy-Art-Scientific-Python/dp/1491922877) *by Juan Nunez-Iglesias, Stefan van der Walt and Harriet Dashnow*
 
-Besides the ones listed above, you may want to check out the [Goodreads list](https://www.goodreads.com/shelf/show/python-scipy) on the subject of "Python+SciPy". Most of the books on this list are about the "SciPy ecosystem", which has NumPy at its core.
+You may also want to check out the [Goodreads list](https://www.goodreads.com/shelf/show/python-scipy) on the subject of "Python+SciPy". Most books there are about the "SciPy ecosystem," which has NumPy at its core.
```

Comment:
minor: is putting the comma within the quotes normal style? looks a little odd to me

Reply:
> minor: is putting the comma within the quotes normal style? looks a little odd to me

I was inconsistent here; the period should be inside "Python+SciPy." too.

Per [Google style guide](https://developers.google.com/style/):
![image](https://user-images.githubusercontent.com/6691888/82750086-dc40e480-9d7b-11ea-9dd4-57b5afac0d6a.png)